src/suites/cts/command_buffer/compute/sampled_texture_clear.spec.ts

```diff
@@ -0,0 +1,75 @@
+export const description = `
```

Comment:
nit: I think we'll end up having a number of tests for resource initialization. How about making a new directory just for that? e.g. `src/suites/cts/resource_init/`, `sampled_texture.spec.ts` or `compute/sampled_texture.spec.ts` depending on whether you think we'll probably want to split files based on pipeline type in the long run.